### ISSUE TITLE
Bootstrap v5: disabled select not grayed out (regression)

### ISSUE THREAD

**Issue by joostdebruijn** [orchidjs/tom-select#425]
**Describe the bug**
When I disable a tom-select element, I expect it would be grayed-out. See the example in the Bootstrap-documentation: https://getbootstrap.com/docs/5.2/forms/select/#disabled. This was reported and fixed earlier: #86, however it now occurs again with Bootstrap 5.2.0 and tom-select 2.1.0.

**To Reproduce**
1. Create a disabled tom-select (either by using the `disabled`-attribute or `disable()` in the Javascript API)
2. The select is not grayed out.

**Expected behavior**
I would expect that it looks grayed out.

**Additional context**
 - OS: Windows  10.0.22621.232
 - Browser: Chrome 
 - Version: 103.0.5060.134
 - Device: Desktop
 - Bootstrap: 5.2.0
 - Tom-select: 2.1.0

**Comment by oyejorge** [comment]
That's annoying, any idea what changed?

**Comment by joostdebruijn** [comment]
Hi @oyejorge. I'm not sure if it was ever fixed: https://codepen.io/joostdeb/pen/wvmqbQm (Bootstrap 5.0.1 - same version as in previous issue with TomSelect 1.7.1 which was the fixed version for issue #89). The weird thing is: in the slow rendering of Codepen the input looks disabled for a while, but later it looks enabled again (while its actually disabled).

I'm not sure now, but I thought I saw it was solved after using 1.7.1, but I can't reproduce that.
